settings.gradle.kts

```diff
@@ -118,7 +118,7 @@ dependencyResolutionManagement {
       version("hapi-version", "0.38.1-allowance-SNAPSHOT")
 
       // Definition of version numbers for all libraries
-      version("pbj-version", "0.5.1")
+      version("pbj-version", "0.5.2")
```

Comment:
Is it really a part of this PR?

Reply:
It's not but this version bump is required to make build work on Windows machines.